dojo-starter/.tool-versions

```diff
@@ -1 +1 @@
-dojo 1.0.2
```

Comment:
we should not change this file, it is just read only
Can check [here](https://github.com/The-Marquis-Gaming/themarquis-ui/blob/1330d181b11b2332a23034e5c75c6a913c34badd/.github/workflows/contracts.yaml#L41C16-L41C24) how to read it